packages/zoe/src/contracts/operaConcertTicket.js

```diff
@@ -68,116 +70,100 @@ export const makeContract = harden(zcf => {
     );
     const ticketsPayment = mint.mintPayment(ticketsAmount);
 
-    const {
-      invite: contractSelfInvite,
-      inviteHandle: contractOfferHandle,
-    } = zcf.makeInvite();
+    let internalTicketSupplyHandle;
+    const internalTicketSupplyOfferHook = offerHandle =>
+      (internalTicketSupplyHandle = offerHandle);
+
+    const contractSelfInvite = zcf.makeInvitation(
```

Comment:
I think the `inviteAnOffer` helper helps both readability via `expected` and debuggability via 
```js
customProperties: {
        inviteDesc: ...,
},
```

That's my preference and what we should typically teach users. But there's nothing wrong with `zcf.makeInvitation` so your call.

Likewise for the `zcf.makeInvitation` below.

Reply:
Ok, thanks! I like zcf.makeInvitation if there aren't any `customProperties` or `expected`. Seemed simpler